bin/Rules.mk

```diff
@@ -1,6 +1,10 @@
 include mk/header.mk
 
 dist_root_$(d)=/ipfs/QmT3CLJKJzWPuN4NAN4LLy69UpKskMF3AuYhXstKdn8V43
+binpostfix :=
```

Comment:
We already have `binpostfix` in this makefile system. You can use `$(?exe)` to access it.